.github/workflows/status-board-update.yml

```
@@ -0,0 +1,140 @@
+
+          ${{ steps.quality-metrics.outputs.summary }}
+
+          ${{ steps.perf-metrics.outputs.summary }}"
+
+          gh issue edit ${{ env.STATUS_BOARD_ISSUE }} --body "$body"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Error Notification
+        if: failure()
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)